components/inbox-mail/service-mail.tsx

```diff
@@ -0,0 +1,88 @@
+import { FetchType, jsonFetchWrapper } from "@/submodules/javascript-functions/basic-fetch";
+import { InboxMailThreadSupportProgressState } from "./types-mail";
+import { convertCamelToSnakeCase } from "@/submodules/javascript-functions/case-types-parser";
+
+const url = `/refinery-gateway/api/v1`;
+
+export function createInboxMailByThread(content: string, onResult: (result: any) => void, recipientIds?: string[], subject?: string, isImportant?: boolean, metaData?: any, threadId?: string, isAdminSupportThread?: boolean) {
+    const body = JSON.stringify({ recipientIds, threadId, subject, content, isImportant, metaData, isAdminSupportThread });
+    jsonFetchWrapper(`${url}/inbox-mail`, FetchType.POST, onResult, body);
```

Comment:
in the other endpoints the url is stored in a const, maybe also here for consistency